components/CreateButton/CreateButton.styled.js

```diff
@@ -1,6 +1,13 @@
-import { styled } from "styled-components";
+import styled from "styled-components";
 
 export const StyledCreateButton = styled.button`
-  background-color: #90ee90;
-  font-size: 13px;
+  margin-top: 50px;
+  width: 115px;
+  height: 32px;
+  border-radius: 13px;
+  border: 1px solid #257336;
+  background: #fff;
+  font-size: 16px;
```

Comment:
This, for example, and of course you already know this, is `1rem`.

Reply:
We will rework this in a seperate refactoring branch! :D